misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -223,3 +227,17 @@ data class CorsConfig(
   /** A comma separated list of HTTP headers that are allowed to be exposed on the client. */
   val exposedHeaders: Array<String> = arrayOf()
 )
+
+data class ConcurrencyLimiterConfig(
+  /** If true, disables automatic load shedding when degraded. */
+  val enabled: Boolean = true,
+
+  /** The algorithm to use for determining concurrency limits. */
+  val strategy: ConcurrencyLimiterStrategy = ConcurrencyLimiterStrategy.VEGAS,
+
+  /** Maximum allowed concurrency limit providing an upper bound failsafe. */
+  val max_concurrency: Int? = null,
+
+  /** Initial limit used by the concurrency limiter. */
```

Comment:
should we document here that this defaults to half of the max thread pool size, if not specified?

Reply:
added documentation of this where the default is (above)!